modules/targets/darwin/options.nix

```diff
@@ -63,6 +73,280 @@ in {
 
       NSAutomaticSpellingCorrectionEnabled =
         mkNullableEnableOption "spelling correction";
+
+      AppleEnableMouseSwipeNavigateWithScrolls = mkOption {
```

Comment:
It's a bit of a hassle to change, but I'd prefer sticking with `mkNullableOption` or `mkNullableEnableOption` for consistency with other options in this file.

Reply:
Agreed. What's in this PR is just a cut and paste from nix-darwin. I may have time next week to normalize, unless someone beats me to it.